### ISSUE TITLE
polkit: Passwordless mount disk partitions for sudoers

### ISSUE THREAD

**Issue by kelebek333** [linuxmint/lmde-5-cinnamon-beta#35]
Hi,

Could you add passwordless mount to partitions like Ubuntu base?

File

`/var/lib/polkit-1/localauthority/10-vendor.d/com.lmde.udisks.pkla`

Content

```
[Mounting, checking, etc. of internal drives]
Identity=unix-group:admin;unix-group:sudo
Action=org.freedesktop.udisks.filesystem-*;org.freedesktop.udisks.drive-ata-smart*;org.freedesktop.udisks2.filesystem-mount-system;org.freedesktop.udisks2.encrypted-unlock-system;org.freedesktop.udisks2.filesystem-fstab;
ResultActive=yes
```

**Comment by clefebvre** [comment]
What package provides this in Ubuntu?

**Comment by kelebek333** [comment]
> What package provides this in Ubuntu?

policykit-desktop-privileges

**Comment by clefebvre** [comment]
Fixed in debian-system-adjustments 2022.03.09. Please close the issue if you're able to test the fix.

**Comment by kelebek333** [comment]
Thanks, it's working.
